### PR TITLE
osd: print log when osd want to kill self

### PR DESCRIPTION
**Pull request by yuyuyu101**
Signed-off-by: Haomai Wang haomai@xsky.com

### CODE REVIEW

**Comment by yuyuyu101** [comment]
we want to know the reason make decision to shutdown, rebind failed or too much failure retries

File: src/osd/OSD.cc [hunk 1]
```diff
@@ -6926,18 +6926,21 @@ void OSD::_committed_osd_maps(epoch_t first, epoch_t last, MOSDMap *m)
 	if (r != 0) {
 	  do_shutdown = true;  // FIXME: do_restart?
           network_error = true;
+          dout(0) << __func__ << " marked down: rebind failed" << dendl;
```
Comment (Aran85) [inline src/osd/OSD.cc]: Maybe print the class of messager will be better?

Comment (yuyuyu101) [inline src/osd/OSD.cc]: for qa, I think developers could know the messenger type from job describe. for user, it should definitely be one type defined in ceph.conf

Comment (tchaikov) [inline src/osd/OSD.cc]: i think it's aways `MOSDMap`?

Comment (yuyuyu101) [inline src/osd/OSD.cc]: yes, message type should always be MOSDMap

**Comment by liewegas** [comment]
retest this please

**Pull request merged**